Loop54.Shared/Model/Request/GetPopularOrRecentEntitiesRequest.cs

```diff
@@ -9,6 +9,10 @@ public class GetPopularEntitiesRequest : GetPopularOrRecentEntitiesRequest
         public GetPopularEntitiesRequest(string behaviorType, string[] entityType, string forUserId) : base(behaviorType, entityType, forUserId)
         {
         }
+
+        public GetPopularEntitiesRequest(string behaviorType, string[] entityType) : base(behaviorType, entityType, Loop54Client.CurrentUserLiteral)
```

Comment:
I think this behavior will be user-surprising: I would generally expect "omitting" a parameter to mean the same as passing null, empty or similar, which in this case means "all user" (global popularity/recency), not "current user".

Reply:
Not having the extra constructors is better than having potentially-confusing constructors, but we can probably get the best of both worlds with static methods:

```
/// <summary>...
public static GetPopularEntitiesRequest ForCurrentUser(...)
```